src/pages/workspace/accounting/qbo/export/QuickbooksOutOfPocketExpenseAccountSelectPage.tsx

```diff
@@ -27,18 +27,40 @@ function QuickbooksOutOfPocketExpenseAccountSelectPage({policy}: WithPolicyConne
 
     const {reimbursableExpensesExportDestination, reimbursableExpensesAccount} = policy?.connections?.quickbooksOnline?.config ?? {};
 
+    const [title, description] = useMemo(() => {
+        let titleText: string | undefined;
+        let descriptionText: string | undefined;
+        switch (reimbursableExpensesExportDestination) {
+            case CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.CHECK:
+                titleText = translate('workspace.qbo.bankAccount');
+                descriptionText = translate('workspace.qbo.bankAccountDescription');
+                break;
+            case CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.JOURNAL_ENTRY:
+                titleText = translate('workspace.qbo.account');
+                descriptionText = translate('workspace.qbo.accountDescription');
+                break;
+            case CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.VENDOR_BILL:
+                titleText = translate('workspace.qbo.accountsPayable');
+                descriptionText = translate('workspace.qbo.accountsPayableDescription');
+                break;
+            default:
```

Comment:
Sorry my OCD 🤦 , can we just add a break if there's no code after default case 🙋